File: 02_구현_기출문제/01_럭키_스트레이트.cpp
#include <iostream>
#include <string>

using namespace std;

int main(void){

    string n;
    getline(cin, n);

    int left = 0, right = 0;;
    for (int i = 0; i < n.size()/2; i++)
    {
        left += n[i] - '0';
    }
    for (int i = n.size()/2; i < n.size(); i++)
    {
        right += n[i] - '0';
    }
    if(left == right){
        cout << "LUCKY";
    }
    else {
        cout << "READY";
    }

    return 0;
}

File: 02_구현_기출문제/01_럭키_스트레이트.py
n = input()
mid = len(n)//2
left = sum(list(map(int, n[:mid])))
right = sum(list(map(int, n[mid:])))

if left == right :
    print("LUCKY")
else :
    print("READY")

'''
(1)
123402

(2)
7755

'''